src/main/java/seedu/teachbook/ui/ClassListPanel.java

```diff
@@ -28,6 +29,10 @@ public ClassListPanel(ObservableList<Class> classList) {
         super(FXML);
         classListView.setItems(classList);
         classListView.setCellFactory(listView -> new ClassListViewCell());
+
+        //TODO: Make it not selectable by mouse click
+        classListView.getSelectionModel().select(0);
+
```

Comment:
This is workable solution I found [here](https://stackoverflow.com/questions/20621752/javafx-make-listview-not-selectable-via-mouse/46186195). You can also try the first answer maybe?
```
classListView.addEventFilter(MouseEvent.MOUSE_PRESSED, event -> {
    // System.out.println(">> Mouse Clicked");
    event.consume();
});
```